pkg/kubelet/cm/dra/manager_test.go

```diff
@@ -408,13 +408,13 @@ func TestPrepareResources(t *testing.T) {
 			description:    "claim doesn't exist",
 			driverName:     driverName,
 			pod:            genTestPod(),
-			expectedErrMsg: "failed to fetch ResourceClaim ",
+			expectedErrMsg: "fetch ResourceClaim ",
 		},
 		{
 			description:    "unknown driver",
 			pod:            genTestPod(),
-			claim:          genTestClaim(claimName, "unknown driver", deviceName, podUID),
-			expectedErrMsg: "plugin name unknown driver not found in the list of registered DRA plugins",
+			claim:          genTestClaim(claimName, "unknown.driver", deviceName, podUID),
```

Comment:
I added the dot to make this more obviously a single driver name and not part of the error message.